src/ui/Facet/ValueElement.ts

```diff
@@ -64,6 +65,17 @@ export class ValueElement {
     this.facetValue.selected = false;
     this.facetValue.excluded = true;
     this.renderer.setCssClassOnListValueElement();
+    var actionCause: IAnalyticsActionCause;
```

Comment:
Why did you move this from the click action ?

It's intentional that the code was in the click action, since we're tracking end user interaction with the search interface, not javascript code that calls exclude or select on values.